pep-8104.rst

```diff
@@ -0,0 +1,152 @@
+PEP: 8104
+Title: 2023 Term Steering Council election
+Author: Ee Durbin <ee@python.org>
+Sponsor: Brett Cannon <brett@python.org>
+Status: Active
+Type: Informational
+Content-Type: text/x-rst
+Created: 08-Nov-2022
+
+
+Abstract
+========
+
+This document describes the schedule and other details of the December
+2022 election for the Python steering council, as specified in
+:pep:`13`. This is the steering council election for the 2023 term
+(i.e. Python 3.12).
+
+
+Election Administration
+=======================
+
+The steering council appointed the
+`Python Software Foundation <https://www.python.org/psf-landing/>`__
+Director of Infrastructure, Ee Durbin, to administer the election.
+
+
+Schedule
+========
+
+There will be a two-week nomination period, followed by a two-week
+vote.
+
+The nomination period shall be: November 14, 2022 through `November 29, 2022 12:00 UTC
```

Comment:
Should the nomination start have a time?

And if November 14, 12:00 UTC to November 29, 12:00 UTC, that's 15 days.

> There will be a two-week nomination period...

Should it be 14 days?

Reply:
It's Nov 14 - Nov 28 AoE which is 14 days.

Reply:
Nov 14-28 AoE is 15 days; Nov 15-28 AoE would be 14 days